### PR TITLE
Fix default currency

### PR DESCRIPTION
**Pull request by madcake**

### CODE REVIEW

File: api/endpoint/tickers.go [hunk 1]
```diff
@@ -88,8 +88,12 @@ func PostTickersHandlerV2(controller controllers.TickersController) func(c *gin.
 			return
 		}
 		request.Ids = removeDuplicates(request.Ids)
+		currency := request.Currency
+		if len(currency) == 0 {
```
Comment (vikmeup) [inline api/endpoint/tickers.go:92]: add test?

**Review by vikmeup — APPROVED** [review]

File: api/endpoint/tickers.go (hunk 1 above)
Comment (madcake) [reply api/endpoint/tickers.go:92]: In Postman.

Comment (vikmeup) [reply api/endpoint/tickers.go:92]: I would suggest to keep such test cases inside codebase, it's external tool

Comment (madcake) [reply api/endpoint/tickers.go:92]: https://github.com/trustwallet/watchmarket/issues/397

**Pull request merged**